### DISCUSSION TITLE
https://plugin-md-enhance.vuejs.press/zh/guide/stylize/attrs.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#850] [Announcements]
# https://plugin-md-enhance.vuejs.press/zh/guide/stylize/attrs.html

你可以使用特殊标记为 Markdown 元素添加属性。 配置 使用 你可以使用语法 {attrs} 来为 Markdown 元素添加属性。 比如，如果你想要一个 id 为 say-hello-world，文字为 Hello World 的二级标题，你可以使用: 如果你想要一个有 full-width Class 的图片，你可以使用: 同时也支持其他属性...

https://plugin-md-enhance.vuejs.press/zh/guide/stylize/attrs.html

**Comment by blankwsh** [comment]
CSS代码应该书写在哪里呢，例子里好像只是添加了类或者id选择器

**Reply by Mister-Hope** [reply]
https://vuejs.press/zh/advanced/cookbook/markdown-and-vue-sfc.html#markdown-and-vue-sfc
